roles/openshift_master_facts/lookup_plugins/openshift_master_facts_default_priorities.py

```diff
@@ -0,0 +1,87 @@
+
+
+class LookupModule(LookupBase):
+    # pylint: disable=too-many-branches
+
+    def run(self, terms, variables=None, zones_enabled=True, **kwargs):
+        if 'openshift' not in variables:
+            raise AnsibleError("This lookup module requires openshift_facts to be run prior to use")
+        if 'master' not in variables['openshift']:
+            raise AnsibleError("This lookup module is meant to be run against an OpenShift master host only")
+
+        if 'openshift_master_scheduler_priorities' in variables:
+            return variables['openshift_master_scheduler_priorities']
+        elif 'scheduler_priorities' in variables['openshift']['master']:
+            return variables['openshift']['master']['scheduler_priorities']
+        else:
+            priorities = [
+                {'name': 'LeastRequestedPriority', 'weight': 1},
+                {'name': 'BalancedResourceAllocation', 'weight': 1},
+                {'name': 'SelectorSpreadPriority', 'weight': 1}
+            ]
+
+            if 'deployment_type' not in variables['openshift']['common']:
+                raise AnsibleError("This lookup module requires that the deployment_type be set")
+
+            deployment_type = variables['openshift']['common']['deployment_type']
+
+            if 'short_version' in variables['openshift']['common']:
+                short_version = variables['openshift']['common']['short_version']
+            elif 'openshift_release' in variables:
+                release = variables['openshift_release']
+                if release.startswith('v'):
+                    short_version = release[1:]
+                else:
+                    short_version = release
+            elif 'openshift_version' in variables:
+                version = variables['openshift_version']
+                short_version = '.'.join(version.split('.')[0:2])
+            else:
+                # pylint: disable=line-too-long
+                raise AnsibleError("Either OpenShift needs to be installed or openshift_release needs to be specified")
+
+            if deployment_type not in ['origin', 'openshift-enterprise']:
+                raise AnsibleError("Unknown deployment_type %s" % deployment_type)
+
+            if deployment_type == 'origin':
+                if short_version not in ['1.1', '1.2', '1.3', '1.4']:
+                    raise AnsibleError("Unknown short_version %s" % short_version)
+            elif deployment_type == 'openshift_enterprise':
+                if short_version not in ['3.1', '3.2', '3.3', '3.4']:
+                    raise AnsibleError("Unknown short_version %s" % short_version)
+
+            if deployment_type == 'openshift-enterprise':
+                # convert short_version to origin short_version
+                short_version = re.sub('^3.', '1.', short_version)
+
+            if short_version == '1.4':
+                priorities.append({'name': 'NodePreferAvoidPodsPriority', 'weight': 10000})
```

Comment:
@jayunit100 agreed, currently they are copied directly from the origin codebase: https://github.com/openshift/origin/blob/release-1.4/vendor/k8s.io/kubernetes/plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go#L113-L206